README.md

```diff
@@ -215,6 +215,51 @@ The pre-commit hooks are run automatically by `git`. They can also be run intera
    ![Sankey diagrame:](docs/Sankey_diagram.png)
 
 
+6. supabase_db_interactions.py
+
+   #### Overview
+   This script demonstrates how to interact with a Supabase database using Python.
+   It utilises the `dotenv` library to load environment variables
+   from a `.env` file and the `supabase-py` library to perform database
+   operations.
+   The script includes functions to write data to and read data from a Supabase
+   table.
+
+   #### Features
+   - Command Line Arguments:
+
+       `--logfile-path`: Specify the path to the logfile. If not provided, it defaults to `supabase/write_to_db.log` in the working directory or the git root.
+   Example:
+   ```
+   python supabase_script.py --logfile-path "/path/to/logfile.log"
+   ```
+   - Environment Variables:
+     - `SUPABASE_URL`: The URL of the Supabase instance.
+     - `SUPABASE_KEY`: The API key for accessing the Supabase instance.
+   Your `.env` file should look like this:
```

Comment:
How about?
"Your `.env` file should look more-or-less like this depending on your actual values for these two environment variables."